src/Encoding/CannotDecodeContent.php

```diff
@@ -15,8 +14,8 @@ public static function jsonIssues(JsonException $previous): self
         return new self('Error while decoding from JSON', 0, $previous);
     }
 
-    public static function invalidBase64String(SodiumException $sodiumException): self
+    public static function invalidBase64String(): self
```

Comment:
Not a BC-Break according to https://github.com/lcobucci/jwt/pull/613#discussion_r544938427